eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -1,13 +1,16 @@
 parameters:
-  - name:  ServiceDirectory
+  - name: ServiceDirectory
     type: string
     default: ''
-  - name:  RunTests
-    type: string
+  - name: RunTests
+    type: boolean
     default: false
-  - name:  RunLiveTests
-    type: string
+  - name: RunLiveTests
+    type: boolean
     default: false
+  - name: RunChangelogCheck
```

Comment:
We should consider a more general "NonShipping" parameter which would turn off validation, testing, releasing etc for things like the tools. Or we could also consider refactoring the existing templates to split out the pieces.

Reply:
Same as above.